### PR TITLE
G4E-6248 application manager rebranding

### PR DESCRIPTION
**Pull request by galin-iliev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Glue42 Floating Toolbar
+# io.Connect Application Manager
```
Comment (arjunah) [inline README.md]: Let's keep the Floating Toolbar name - io.Connect Floating Toolbar

File: app-config.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "title": "AppManager-Embedded",
   "type": "window",
-  "name": "glue42-application-manager",
+  "name": "connect-application-manager",
```
Comment (arjunah) [inline app-config.json]: io-connect-toolbar

File: app-config.json [hunk 3]
```diff
@@ -30,7 +30,7 @@
     },
     "taskbar": {
       "allowPin": true,
-      "appId": "glue42-enterprise-appManager"
+      "appId": "connect-desktop-application-manager"
```
Comment (arjunah) [inline app-config.json]: io-connect-toolbar

File: index.html [hunk 4]
```diff
@@ -7,13 +7,15 @@
       href="./assets/t42bootstrap.bundle.css"
     />
     <link rel="stylesheet" type="text/css" href="scss/app.css" />
-    <title>Glue42 Toolbar</title>
+    <title>io.Connect Application Manager</title>
```
Comment (arjunah) [inline index.html]: io.Connect Floating Toolbar

**Review by arjunah — CHANGES_REQUESTED** [review]

**Comment by galin-iliev** [comment]
@arjunah all requested changes were implemented.

**Review by arjunah — APPROVED** [review]

**Pull request merged**